src/Views/AppInfoView.vala

```diff
@@ -227,20 +227,22 @@ namespace AppCenter.Views {
             if (!package.is_local) {
                 var download_button_grid = new Gtk.Grid ();
                 download_button_grid.orientation = Gtk.Orientation.VERTICAL;
-                download_button_grid.valign = Gtk.Align.END;
+                download_button_grid.valign = Gtk.Align.START;
                 download_button_grid.halign = Gtk.Align.END;
                 download_button_grid.row_spacing = 3;
+                download_button_grid.margin_top = 40;
```

Comment:
I'm not a huge fan of magic numbers. Where is this value from? Can we instead do something like get the allocation of a widget we're trying to align to or use a size group or something?

Reply:
I understand. I needed to Start align the Download grid so it wouldn't move because of the disappearance of the size label. I then needed to drop the grid down to visually align it with the Package name and version. See:
![screenshot from 2018-10-11 13 45 25-line](https://user-images.githubusercontent.com/523210/46801958-5580f480-cd5c-11e8-8212-4bf7aab59499.png)

If there is a better way I'd love to try.